<p>[Peer Evaluated Publications [112: 53 Journals, 54 Proceedings, 5 Book
chapters]]{.smallcaps}</p>
<blockquote>
<p>[Refereed Journal Articles [52 articles]]{.smallcaps}</p>
</blockquote>
<ol>
<li>
<p>Alexey Markin, Sanket E. Wagle, Tavis, Anderson, Oliver Eulenstein.
RF-Net 2: Fast Inference of Virus Reassortment and Hybridization
Networks, <em>Bioinformatics</em> accepted 2022.</p>
</li>
<li>
<p>P. Górecki, O. Eulenstein, and J. Tiuryn, &quot;The Unconstrained
Diameters of the Duplication-Loss Cost and the Loss Cost,&quot;
in <em>IEEE/ACM Transactions on Computational Biology and
Bioinformatics</em>, vol. 18, no. 6, pp. 2125-2135, 2021.</p>
</li>
<li>
<p>Alexey Markin and Oliver Eulenstein. <strong>Quartet-Based Inference
Methods are Statistically Consistent Under the Unified
Duplication-Loss-Coalescence Model. btab414,
<em>Bioinformatics</em>, 2021.</strong></p>
</li>
<li>
<p>Jarosław Paszek, Alexey Markin, Paweł Górecki, Oliver Eulenstein.
Taming the Duplication-Loss-Coalescence Model with Integer Linear
Programming. Journal of Computational Biology 28(8), pages 758 --
773, 2021.</p>
</li>
<li>
<p>Pawel Tabaszewski, Paweł Górecki, Alexey Markin, Tavis Anderson,
Oliver Eulenstein. Consensus of all Solutions for Intractable
Phylogenetic Tree Inference. <em>IEEE/ACM Trans. Comput. Biology
Bioinform 18(1)</em>, pages 149 -- 161, 2021.</p>
</li>
<li>
<p>Huy N. Nguyen, Alexey Markin, Iddo Friedberg, Oliver Eulenstein.
Finding orthologous gene blocks in bacteria: the computational
hardness of the problem and novel methods to address it.
<em>Bioinformatics</em> 36 (Supplement_2), i668-i674, 2020.</p>
</li>
<li>
<p>Paweł Górecki, Alexey Markin, Oliver Eulenstein. Exact Median-tree
Inference for Unrooted Reconciliation Costs. <em>BMC Evol Biol</em> 20
(136), i668 -- i674, 2020.</p>
</li>
<li>
<p>**Paweł Górecki, Agnieszka Mykowieckaa, Jarosław Paszek, Oliver
Eulenstein. Mathematical properties of the gene duplication cost.
<em>Discrete Applied Mathermatics</em>. Volume 258, Pages 114 -- 122, 2019.
**</p>
</li>
<li>
<p><strong>Huy Nguyen, Ashish Jain, [Oliver Eulenstein](javascript:;), [Iddo
Friedberg](javascript:;). Tracing the Ancestry of Operons in
Bacteria,
btz053, <a href="https://doi.org/10.1093/bioinformatics/btz053">https://doi.org/10.1093/bioinformatics/btz053</a>, 2019.</strong></p>
</li>
<li>
<p>Alexey Markin and Oliver Eulenstein. Cophenetic Median Trees.
<em>IEEE/ACM Trans. Comput. Biology Bioinform</em> 16(5), 1459 -- 1470,
September/October 2019.
<a href="https://ieeexplore.ieee.org/document/8464248?source=authoralert">10.1109/TCBB.2018.2870173</a></p>
</li>
<li>
<p>Paweł Górecki and Oliver Eulenstein. Bijective Diameters of Gene
Tree Parsimony Costs. <em>IEEE/ACM Trans. Comput. Biology Bioinform.</em>
(Volume: PP, Issue: 99) 2017.</p>
</li>
<li>
<p>Alexey Markin and Oliver Eulenstein. Efficient Local Search for
Euclidean Path-Difference Median Trees. <em>IEEE/ACM Trans. Comput.
Biology Bioinform</em>, (Volume: PP, Issue: 99) 2017.</p>
</li>
<li>
<p>Alexey Markin and Oliver Eulenstein. <strong>Computing Manhattan
Path-Difference Median Trees: A Practical Local Search Approach.</strong>
<em>IEEE/ACM Trans. Comput. Biology Bioinform</em>, (Volume: PP, Issue: 99)
2017.</p>
</li>
<li>
<p>Jucheol Moon and Oliver Eulenstein. Synthesizing Large-scale Species
Trees using the Strict Consensus Approach. <em>Journal of
Bioinformatics and Computational Biology</em>, 15(3):1740002, 2017.</p>
</li>
<li>
<p>Paweł Górecki, Jarosław Paszek, Oliver Eulenstein. Unconstrained
Diameters for Deep Coalescence. <em>IEEE/ACM Trans. Comput. Biology
Bioinform</em>, 14(5), 1002 -- 1012, 2017.</p>
</li>
<li>
<p>Jucheol Moon, Harris Lin, Oliver Eulenstein. Consensus Properties
and their Large-Scale Applications for the Gene Duplication Problem.
<em>Journal of Bioinformatics and Computational Biology</em>,
14(03):1642005, 2016.</p>
</li>
<li>
<p>Paweł Górecki and Oliver Eulenstein. Gene Tree Diameter for Deep
Coalescence. <em>IEEE/ACM Trans. Comput. Biology Bioinform. 12(1),
155-165, 2015.</em></p>
</li>
<li>
<p>Paweł Górecki and Oliver Eulenstein. Refining Discordant Gene Trees.
<em>BMC Bioinformatics.</em> 15(Suppl 13):S3, 2014.</p>
</li>
<li>
<p>Paweł Górecki and Oliver Eulenstein. Maximizing Deep Coalescence
Cost. <em>IEEE/ACM Trans. Comput. Biology Bioinform.
11(1):231-242</em>, 2014.</p>
</li>
<li>
<p>Paweł Górecki and Oliver Eulenstein. DrML: probabilistic modeling of
gene duplications*. J Comput Biol*, 21(1):89-98, 2014.</p>
</li>
<li>
<p>Mukul S. Bansal, and Oliver Eulenstein. Algorithms for Genome-Scale
Phylogenetics Using Gene Tree Parsimony, <em>IEEE/ACM Trans. Comput.
Biology Bioinform.</em> 10(4):939-56, <em>2013.</em></p>
</li>
<li>
<p>André Wehe, J. Gordon Burleigh, and Oliver Eulenstein. Efficient
Algorithms for Knowledge-Enhanced Supertree and Supermatrix
Phylogenetic Problems. <em>IEEE/ACM Trans. Comput. Biology Bioinform,</em>
10(6):1432-1441, 2013.</p>
</li>
<li>
<p>Paweł Górecki, Oliver Eulenstein, and J. Tiuryn. Unrooted Tree
Reconciliation: A Unified Approach. <em>IEEE/ACM Trans. Comput. Biology
Bioinform 10(2):522-536, 2013.</em></p>
</li>
<li>
<p>Wen-Chieh Chang, Paweł Górecki and Oliver Eulenstein*.* Exact
solutions for species tree inference from discordant gene trees.
<em>Journal of Bioinformatics and Computational Biology</em>
11(5):1342005, 2013.</p>
</li>
<li>
<p>Paweł Górecki and Oliver Eulenstein. Algorithms: Simultaneous
Error-Correction and Rooting for Gene Tree Reconciliation and the
Gene Duplication Problem. <em>BMC Bioinformatics 13 (Suppl 10),
S14, 2012.</em></p>
</li>
<li>
<p>Wen-Chieh Chang, Sudheer Vakati, Roland Krause and Oliver
Eulenstein. Exploring Biological Interaction Networks with Tailored
Weighted Quasi-Bicliques. <em>BMC Bioinformatics 13 (Suppl
10):S12, 2012.</em></p>
</li>
<li>
<p>Ruchi Chaudhary, J. Gordon Burleigh, and Oliver Eulenstein.
Efficient Algorithms for Rapid Error Correction for Gene Tree
Reconciliation using Gene Duplications, Gene Duplication and Loss,
and Deep Coalescence. <em>BMC Bioinformatics 13 (Suppl 10):S11, 2012.</em></p>
</li>
<li>
<p>Harris T. Lin, J. Gordon Burleigh and Oliver Eulenstein. Consensus
Properties for the Deep Coalescence Problem and Their Application
for Scalable Tree Search. <em>BMC Bioinformatics, 13 (Suppl
10):S12, 2012.</em></p>
</li>
<li>
<p>Paweł Górecki and Oliver Eulenstein. Maximum likelihood models and
algorithms for gene tree evolu­tion with duplications and losses.
<em>APBC 2011; BMC Bioinformatics,</em> 12(Suppl 1):S15, 2011.</p>
</li>
<li>
<p>Wen-Chieh Chang, Gordon J Burleigh, David F. Fernández-Baca, and
Oliver Eulenstein. An ILP solution for the gene duplication problem.
<em>APBC 2011; BMC Bioinformatics,</em> 12(Suppl 1):S14, 2011.</p>
</li>
<li>
<p>J. Gordon Burleigh, Mukul S. Bansal, Oliver Eulenstein, Stephanie
Hartmann, André Wehe, and Todd J. Vi­sion. Genome-scale
phylogenetics: Inferring the plant tree of life from 18,896
discordant gene trees. <em>Systematic Biology,</em> 60(2):117-125, 2011.</p>
</li>
<li>
<p>John Wiedenhoeft, Roland Krause, and Oliver Eulenstein. The Plexus
Model for the Inference of Ances­tral Multi-Domain Proteins,
<em>IEEE/ACM Trans. Comput. Biology Bioinform.</em>, 8(4):890-901, 2011.</p>
</li>
<li>
<p>Ruchi Chaudhary, Mukul S. Bansal, André Wehe, David Fernández-Baca,
and Oliver Eulenstein. iGTP: A software package for large-scale gene
tree parsimony analysis. <em>BMC Bioinformatics,</em> 11:574, 2010.</p>
</li>
<li>
<p>Mukul S. Bansal, J. Gordon Burleigh, Oliver Eulenstein, and David
Fernández-Baca. Robinson-Foulds super­trees. <em>Algorithms in
Molecular Biology,</em> 5(1):18, 2010.</p>
</li>
<li>
<p>André Wehe, Wen-Chieh Chang, Oliver Eulenstein, and Srinivas Aluru.
A scalable parallelization of the gene duplication problem. <em>Journal
of Parallel and Distributed Computing,</em> 70:237-244, 2010.</p>
</li>
<li>
<p>Mukul S. Bansal, J. Gordon Burleigh, and Oliver Eulenstein.
Efficient genome-scale phylogenetic analy­sis under the
duplication-loss and deep coalescence cost models. <em>APBC 2010; BMC
Bioinformat­ics</em>,11(Suppl 1):S42, 2010.</p>
</li>
<li>
<p>Mukul S. Bansal, Oliver Eulenstein, and André Wehe. The
gene-duplication problem: Near linear time algo­rithms for NNI-based
local searches. 6 <em>IEEE/ACM Trans. Comput. Biology Bioinform.</em>,
(2):221-231, 2009.</p>
</li>
<li>
<p>Harris T. Lin, J. Gordon Burleigh, and Oliver Eulenstein. Triplet
supertree heuristics for the tree of life. <em>APBC 2009; BMC
Bioinformatics</em>, 10(S-1), 2009.</p>
</li>
<li>
<p>J. Gordon Burleigh, Mukul S. Bansal, Andre Wehe, and Oliver
Eulenstein. Locating large-scale gene duplica­tion events through
reconciled trees: implications for identifying ancient polyploidy
events in plants. <em>J Comput Biol</em>, 16(8): 1071 -- 83, 2009.</p>
</li>
<li>
<p>Cécile Ané, Oliver Eulenstein, Raul Piaggio-Talice, and Michael
Sanderson. Groves of phylogenetic trees. <em>Annals of Combinatorics</em>,
13(2): 139 --167, 2009.</p>
</li>
<li>
<p>André Wehe, Mukul S. Bansal, J. Gordon Burleigh, and Oliver
Eulenstein. Duptree: a program for large-scale phylogenetic analyses
using gene tree parsimony. <em>Bioinformatics</em>, 24(13):1540-1541, 2008.</p>
</li>
<li>
<p>Mukul S. Bansal and Oliver Eulenstein. The multiple gene duplication
problem revisited. <em>ISMB; Bioinfor­matics,</em> 24(13):i132-8, 2008.</p>
</li>
<li>
<p>Mukul S. Bansal and Oliver Eulenstein. An Ω(<em>n</em>^2^/ log n) speed-up
of TBR heuristics for the gene-duplica­tion problem. <em>IEEE/ACM
Trans. Comput. Biology Bioinform.</em>, 5(4): 514 --524, 2008. A
preliminary version of this paper appeared in WABI 2007.</p>
</li>
<li>
<p>Duhong Chen, Oliver Eulenstein, David Fernández-Baca, and J. Gordon
Burleigh. Improved heuristics for minimum-flip supertree
construction. <em>Evol Bioinform Online</em>, 2: 391 -- 400, 2006.</p>
</li>
<li>
<p>Duhong Chen, Oliver Eulenstein, David Fernández-Baca, and Michael J.
Sanderson. Minimum-flip super­trees: Complexity and algorithms.
<em>IEEE/ACM Trans. Comput. Biology Bioinform.</em>, 3(2):165-173, 2006.</p>
</li>
<li>
<p>Changhui Yan, J. Gordon Burleigh, and Oliver Eulenstein. Identifying
optimal incomplete phylogenetic data sets from sequence databases.
<em>Mol Phylogenet Evol,</em> 35(3):528-35, 2005.</p>
</li>
<li>
<p>Mark Wilkinson, James A. Cotton, Chris Creevey, Oliver Eulenstein,
Simon R. Harris, Francois-Joseph Lapointe, Claudine Levasseur,
James O. McInerney, Davide Pisani, and Joseph L. Thorley. The shape
of supertrees to come: tree shape related properties of fourteen
supertree methods. <em>Syst Biol</em>, 54(3):419-31, 2005.</p>
</li>
<li>
<p>Oliver Eulenstein, Duhong Chen, J. Gordon Burleigh, David
Fernández-Baca, and Michael J. Sanderson. Performance of flip
supertree construction with a heuristic algorithm. <em>Syst Biol</em>,
53(2):299-308, 2004.</p>
</li>
<li>
<p>Duhong Chen, Oliver Eulenstein, and David Fernández-Baca. Rainbow: a
toolbox for phylogenetic super­tree construction and analysis.
<em>Bioinformatics</em>, 20(16):2872-2873, 2004.</p>
</li>
<li>
<p>Michael J. Sanderson, Amy C Driskell, Richard H Ree, Oliver
Eulenstein, and Sasha Langley. Obtaining maximal concatenated
phylogenetic data sets from large sequence databases. <em>Mol Biol
Evol</em>, 20(7):1036-42, Jul 2003.</p>
</li>
<li>
<p>Oliver Eulenstein, Boris Mirkin, and Martin Vingron.
Duplication-based measures of difference between gene and species
trees. <em>Journal of Computational Biology</em>, 5(1):135-148, 1998.</p>
</li>
<li>
<p>Oliver Eulenstein and Martin Vingron. On the equivalence of two tree
mapping measures. <em>Discrete Ap­plied Mathematics</em>,
88(1-3):103-128, 1998.</p>
</li>
<li>
<p>Yan P. Yuan, Oliver Eulenstein, Martin Vingron, and Peer Bork.
Towards detection of orthologues in se­quence databases.
<em>Bioinformatics</em>, 14(3):285-289, 1998.</p>
</li>
</ol>
<p>[Proceedings from Major Conferences [54 articles]]{.smallcaps}</p>
<blockquote>
<p>[(In Computer Science refereed conference publications and journal
publications are seen to be equally important)]{.smallcaps}</p>
</blockquote>
<ol>
<li>
<p>Alvin Chon, Huang Xiaoqiu, Oliver Eulenstein, Ali Jannesari.
Scalable and Extensible Robinson-Foulds for Comparative
Phylogenetics. 21^st^ IEEE International Workshop on High
Performance Computational Biology, Lyon France (virtual), May
30, 2022.</p>
</li>
<li>
<p>Adithya Kulkarni, Nasim Sabetpour, Alexey Markin, Oliver Eulenstein,
Qin Li. CPTAM: Constituency Parse Tree Aggregation Method. SIAM
International Conference on Data Mining (SDM 2022). Accepted.</p>
</li>
<li>
<p>Jarosław Paszek, Oliver Eulenstein, Paweł Górecki, Novel genomic
duplication models through integer linear programming. <em>BCB '21:</em>
<em>Proceedings of the 12th ACM Conference on Bioinformatics,
Computational Biology, and Health Informatics.</em> No 16, pp 1 --
11, 2021.</p>
</li>
<li>
<p>Javad Ansarifar, Alexey Markin, Pawel Gorecki and Oliver Eulenstein.
Integer Linear Programming Formulation for the Unified
Duplication-Loss-Coalescence Model.<em>15^th^ International Symposium
on Bioinformatics Research and Applications (ISBRA)</em>, Moscow,
Russia, December 1 -- 4, LNCS 1230, pp. 229 -- 242, <strong>2020.</strong></p>
</li>
<li>
<p>Alexey Markin, Tavis K. Anderson, Venkata Sai Krishna Teja Vadali,
Oliver Eulenstein. Robinson-Foulds Reticulation Networks.
<strong><em>Proceedings of the</em> <em>10th ACM Conference on Bioinformatics,
Computational Biology, and Health Informatics (ACM BCB),</em> Niagara
Falls, Sept. 8 -- 10, pp. 77</strong> -- 86, <strong>2019.</strong></p>
</li>
<li>
<p>Alexey Markin and Oliver Eulenstein. Consensus Clusters in
Robinson-Foulds Reticulation Networks. Workshop on Algorithms in
Bioinformatics (WABI) Niagara Falls, NY, Sept. 8-10, <strong>LIPIcs, Vol.
143, pp. 12:1--12:12,</strong> 2019.</p>
</li>
<li>
<p>Paweł Górecki, Alexey Markin, Oliver Eulenstein. Feasibility
Algorithms for the Duplication-Loss Cost. <em>25^th^ International
Computing and Combinatorics Conference (COCOON)</em>, Xian (China), Jul.
29-31, LNCS vol 11653, pp. 206 - 218, 2019.</p>
</li>
<li>
<p>Jucheol Moon, Oliver Eulenstein. The Cluster Affinity Distance for
Phylogenies. <strong><em>15^th^ International Symposium on Bioinformatics
Research and Applications (ISBRA)</em>, Barcelona, Spain, June 3 - 6,</strong>
LNCS 11490, pp. 52 -- 64, <strong>2019.</strong></p>
</li>
<li>
<p>Paweł Tabaszewski, Paweł Górecki, Oliver Eulenstein. Phylogenetic
Consensus for Exact Median Trees. <strong><em>Proceedings of the</em> <em>9th ACM
Conference on Bioinformatics, Computational Biology, and Health
Informatics (ACM BCB),</em> Washington DC, Aug. 29 -- Sept. 1, pp. 366
-- 375, 2018.</strong></p>
</li>
<li>
<p>Paweł Górecki, Alexey Markin, Oliver Eulenstein. Cophenetic
Distances: A Near-Linear Time Algorithmic Framework, <em>24^th^
International Computing and Combinatorics Conference (COCOON)</em>,
Quing Dao, Jul. 2-4, LNCS vol 10976, pp. 168 -- 179, 2018.</p>
</li>
<li>
<p>Alexey Markin, Venkata Sai Krishna Teja Vadali and Oliver
Eulenstein. Solving the Gene Duplication Feasibility Problem in
Linear Time. <em>24^th^ International Computing and Combinatorics
Conference (COCOON)</em>, Quing Dao, Jul. 2-4, LNCS vol 10976, pp. 378
-- 390, 2018.</p>
</li>
<li>
<p>Jucheol Moon and Oliver Eulenstein. Cluster Matching Distance for
Rooted Phylogenetic Trees. <strong><em>13^th^ International Symposium on
Bioinformatics Research and Applications (ISBRA)</em>, Bejing, China,
June 8 -- June 11,</strong> LNCS vol 10847, pp. 321 -- 332, <strong>2018.</strong></p>
</li>
<li>
<p>Alexey Markin and Oliver Eulenstein. Cophenetic Median Trees Under
the Manhattan Distance*. <strong>Proceedings of the</strong>* <strong><em>8th ACM
Conference on Bioinformatics, Computational Biology, and Health
Informatics (ACM BCB),</em> Boston, Aug. 20-23, pp. 194 -- 202, 2017.</strong></p>
</li>
<li>
<p>Jucheol Moon and Oliver Eulenstein. Synthesizing Species Trees from
Unrooted Gene Trees: A Parameterized Approach. <strong>Proceedings of the
<em>8th ACM Conference on Bioinformatics, Computational Biology, and
Health Informatics (ACM BCB),</em> Boston, Aug. 20-23, pp. 253 --
262, 2017.</strong></p>
</li>
<li>
<p>Paweł Górecki**, Alexey Markin, Agnieszka Mykowiecka, Jaroslaw
Paszek and Oliver Eulenstein. Phylogenetic Tree Reconciliation: Mean
Values for Fixed Gene Trees. <em>12^th^ International Symposium on
Bioinformatics Research and Applications (ISBRA)</em>, Honolulu, USA,
May 29 -- June 2,** LNCS 10330, pp. 234 -- 245, <strong>2017.</strong></p>
</li>
<li>
<p><strong>Alexey Markin and Oliver Eulenstein. Manhattan Path-Difference
Median Trees, Proceedings of the <em>7th ACM Conference on
Bioinformatics, Computational Biology, and Health Informatics (ACM
BCB),</em> Seattle, USA, Oct. 2-5., pp. 404 -- 413, 2016.</strong></p>
</li>
<li>
<p>Jucheol Moon and Oliver Eulenstein. <strong>Robinson-Foulds Median Trees:
A Clique-based Heuristic, Proceedings of the <em>7th ACM Conference on
Bioinformatics, Computational Biology, and Health Informatics (ACM
BCB),</em> Seattle, Oct. 2-5., pp. 374 -- 383, 2016.</strong></p>
</li>
<li>
<p>Jucheol Moon, Iddo Friedberg, Oliver Eulenstein. Highly Bi-Connected
Subgraphs for Computational Protein Function Annotation, 22^nd^
International Computing and Combinatorics Conference (COCOON), Ho
Chi Minh city, Aug. 2-4, LNCS vol 9797, pp. 573 -- 548, 2016.</p>
</li>
<li>
<p><strong>Alexey Markin and Oliver Eulenstein.</strong> Path-Difference Median
Trees, <em>12^th^ International Symposium on Bioinformatics Research
and Applications (ISBRA)</em>, Minks, Belarus, June 5 -- 8, LNCS 9683,
pp. 211 -- 223, 2016.</p>
</li>
<li>
<p>Jucheol Moon and Oliver Eulenstein. Synthesizing Large-Scale Species
Trees using Guidance Trees, 8th International Conference on
Bioinformatics and Computational Biology (BICoB), Las Vegas, 4-6
April, pp 103 -- 136, Editor: F. Saeed and Nurit Haspel, Curran
Association, Inc. 2016. <strong>'Best Paper Finalist'</strong> (there were three
finalist papers)</p>
</li>
<li>
<p>Harris T. Lin, Jucheol Moon, Oliver Eulenstein. Consensus Properties
of the Gene Duplication Problem for Enhanced Phylogenetic Inference,
7th International Conference on Bioinformatics and Computational
Biology (BICoB), Editor: F. Saeed and Nurit Haspel, Curran
Association, Inc. 2015: 131-136.</p>
</li>
<li>
<p>Paweł Górecki, Jarosław Paszek, Oliver Eulenstein. Unconstrained
Gene Tree Diameters for Deep Coalescence, ACM-BCB 2014.</p>
<p><strong>Awarded 'Best Paper'</strong></p>
</li>
<li>
<p>Paweł Górecki, Jarosław Paszek, Oliver Eulenstein. Duplication Cost
Diameters. ISBRA 2014, LNCS 8492: 212-223 <em>(invited abstract for a
full version in the International Journal of Data Mining and
Bioinformatics).</em></p>
</li>
<li>
<p>Zhipeng Cai, Oliver Eulenstein, Daniel Janies, Daniel Schwartz
(Eds.): Bioinformatics Research and Applications, 9th International
Symposium, ISBRA 2013, Charlotte, NC, USA, May 20-22, 2013.
Proceedings. Lecture Notes in Computer Science 7875, Springer 2013.</p>
</li>
<li>
<p>Paweł Górecki and Oliver Eulenstein. Algorithms for Unrooted Gene
Trees with Polytomies. <em>Short Abstract Proceedings of ISBRA:86-95,
2013 (invited abstract for a full version in BMC Genetics)</em>.</p>
</li>
<li>
<p>Bryan Shrader, <a href="http://www.informatik.uni-trier.de/~ley/pers/hd/w/Wehe:Andr=eacute=.html">André
Wehe</a>,
Oliver Eulenstein, Mathematical Properties and Normalization of the
Gene Duplication Cost. <em>Short Abstract Proceedings of ISBRA:92-96,
2013 (invited abstract for a full version in BMC Genetics)</em>.</p>
</li>
<li>
<p>Wen-Chieh Chang, André Wehe, Paweł Górecki, and Oliver Eulenstein.
Exact Solutions for Classical Gene Tree Parsimony Problems.
<em>BICoB 2013.</em></p>
</li>
<li>
<p>Sabine Baumann and Oliver Eulenstein. Who pulls the strings behind
the scenes? Analyzing power structures of international media
finance networks with innovative graph-based methods. Internet,
Politics, Policies (IPP) 2012: Big Data, Big Challenges?
http://microsites.oii.ox.ac.uk/ipp2012.</p>
</li>
<li>
<p><a href="http://www.informatik.uni-trier.de/~ley/pers/hd/w/Wehe:Andr=eacute=.html">André
Wehe</a>, <a href="http://www.informatik.uni-trier.de/~ley/pers/hd/b/Burleigh:John_Gordon.html">J.
Gordon
Burleigh</a>, Oliver
Eulenstein: Algorithms for Knowledge-Enhanced Supertrees. <em><a href="http://www.informatik.uni-trier.de/~ley/db/conf/isbra/isbra2012.html#WeheBE12">ISBRA
2012</a>,</em>
LNCS7292: 263-274.</p>
</li>
<li>
<p><a href="http://www.informatik.uni-trier.de/~ley/pers/hd/g/G=oacute=recki:Pawel.html">Paweł
Górecki</a>, <a href="http://www.informatik.uni-trier.de/~ley/pers/hd/b/Burleigh:John_Gordon.html">J.
Gordon
Burleigh</a>, Oliver
Eulenstein. GTP Supertrees from Unrooted Gene Trees: Linear Time
Algorithms for NNI Based Local Searches. <em>ISBRA 2012</em>, LNCS 7292:
102-104.</p>
</li>
<li>
<p><a href="http://www.informatik.uni-trier.de/~ley/pers/hd/g/G=oacute=recki:Pawel.html">Paweł
Górecki</a>
and Oliver Eulenstein. Deep Coalescence Reconciliation with Unrooted
Gene Trees: Linear Time Algorithms. <em>COCOON 2012</em>, LNCS 7434:
531-542.</p>
</li>
<li>
<p><a href="http://www.informatik.uni-trier.de/~ley/pers/hd/g/G=oacute=recki:Pawel.html">Paweł
Górecki</a>
and Oliver Eulenstein: A Robinson-Foulds Measure to Compare Unrooted
Trees with Rooted Trees. <a href="http://www.informatik.uni-trier.de/~ley/db/conf/isbra/isbra2012.html#GoreckiE12"><em>ISBRA
2012</em></a>,
LNCS 7292: 115-126.</p>
</li>
<li>
<p>Paweł Górecki and Oliver Eulenstein. A linear time algorithm for
error-corrected reconciliation of unrooted gene trees. <em>ISBRA 2011,
LNCS6674</em>: 148-159.</p>
</li>
<li>
<p>Wen-Chieh Chang, Sudheer Vakati, Roland Krause and Oliver
Eulenstein. Mining Biological Interaction Networks using Weighted
Quasi-Bicliques. <em>ISBRA 2011,</em> <em>LNCS6674</em>:428-439.</p>
</li>
<li>
<p>Ruchi Chaudhary, J. Gordon Burleigh, and Oliver Eulenstein.
Algorithms for Rapid Error Correction for the Gene Duplication
Problem. <em>ISBRA 2011, LNCS6674</em>: 227-239.</p>
</li>
<li>
<p>Sabine Baumann and Oliver Eulenstein. Life in the Fast Lane:
Analysing corporate communication networks with innovative
graph-based methods to prepare for crisis situations&quot;, in: Centre
for Corporate Communication (Hrsg.) Communicating Crisis in an Age
of Complexity, Conference Proceedings #1 of the 2nd International
Conference on Crisis Communication at the Beginning of the 21st
Century, Aarhus University, Aarhus (DK), S. 20-44, 2011.</p>
</li>
<li>
<p>Oliver Eulenstein. Invited: Reconciled gene trees and their
applications. ICCABS, IEEE Computer Society, 2011.</p>
</li>
<li>
<p>Harris T. Lin, J. Gordon Burleigh and Oliver Eulenstein. The Deep
Coalescence Consensus Tree Problem is Pareto on Clusters. <em>ISBRA
2011,</em> <em>LNCS6674</em>: 172-183.</p>
</li>
<li>
<p>J. Gordon Burleigh, Mukul S. Bansal, Oliver Eulenstein, and Todd J.
Vision. Inferring Species Trees From Gene Duplication Episodes. In
<em>Proc. of ACM Conf. on Bioinformatics and Computational Biology
(ACM-BCB),</em> pages 198-203, 2010.</p>
</li>
<li>
<p>Homilius Max, J. Gordon Burleigh, and Oliver Eulenstein.
Triplet-supertrees constructed from minimum triplet presentations.
In <em>Proceedings of the ISCA 2nd International Conference on
Bioinformatics and Computational Biology, BICoB (Al-Mubaid Hisham,
ed.),</em> pages 127-132, 2010.</p>
</li>
<li>
<p>John Wiedenhoeft, Roland Krause, and Oliver Eulenstein. Inferring
Evolutionary Scenarios for Protein Do­main Compositions. <em>ISBRA
2010,</em> LNCS6053:179-190, 2010.</p>
</li>
<li>
<p>Mukul S. Bansal, Wen-Chieh Chang, Oliver Eulenstein, and David
Fernández-Baca. Generalized binary tan­glegrams: Algorithms and
applications. In <em>BICoB; Lecture Notes in Computer Science
(Sanguthevar Rajasekaran, ed.),</em> 5462:114-125, 2009.</p>
</li>
</ol>
<blockquote>
<p><strong>Awarded 'Best Paper'</strong></p>
</blockquote>
<ol start="43">
<li>
<p>J. Gordon Burleigh, Mukul S. Bansal, André Wehe, and Oliver
Eulenstein. Locating multiple gene duplica­tions through reconciled
trees. In <em>RECOMB, Lecture Notes in Computer Science (Martin Vingron
and Limsoon Wong, ed.)</em>, 4955:273-284, 2008.</p>
</li>
<li>
<p>Mukul S. Bansal and Oliver Eulenstein. The gene-duplication problem:
Near-linear time algorithms for NNI based local searches. In <em>ISBRA;
Lecture Notes in Computer Science (Ion I. Mandoiu, Raj Sunder­raman,
and Alexander Zelikovsky, ed.)</em>, 4983:14-25. Springer, 2008.</p>
</li>
<li>
<p>Mukul S. Bansal, Oliver Eulenstein. An Ω(<em>n</em>^2^/ log n) speed-up of
TBR heuristics for the gene-duplication problem. In <em>WABI 2007;
Lecture Notes in Computer Science (Raaele Giancarlo and Sridhar
Hannenhalli, ed.),</em> 4645:124-135, 2007.</p>
</li>
<li>
<p>Mukul S. Bansal, J. Gordon Burleigh, Oliver Eulenstein, and André
Wehe. Heuristics for the gene-duplica­tion problem: An Ω(n) speed-up
for the local search. In <em>RECOMB; Lecture Notes in Computer Sci­ence
(Terence P. Speed and Haiyan Huang, ed.),</em> 4453:238-252.
Springer, 2007.</p>
</li>
<li>
<p>Wen-Chieh Chang and Oliver Eulenstein. Reconciling gene trees with
apparent polytomies. In <em>COCOON; Lecture Notes in Computer Science
(Danny Z. Chen and D. T. Lee, ed.),</em> 4112:235-244, 2006.</p>
</li>
<li>
<p>Duhong Chen, Lixia Diao, Oliver Eulenstein, David Fernández-Baca,
and Michael J. Sanderson. Flipping: A supertree construction method.
In <em>Bioconsensus, DIMACS Series in Discrete Mathematics and
Theo­retical Computer Science (M. F. Janowitz, ed.)</em>, 37:135-169,
American Mathematical Society, Provi­dence, RI, 2003.</p>
</li>
<li>
<p>J. Schonfeld, Oliver Eulenstein, Kent Vander Velden, and Gavin J. P.
Naylor. Investigating evolutionary lines of least resistance using
the inverse protein-folding problem. In <em>Pacific Symposium on
Biocompu­ting</em>, pages 613-624, 2002.</p>
</li>
<li>
<p>Duhong Chen, Oliver Eulenstein, David Fernández-Baca, and Michael
Sanderson. Supertrees by flip­ping. In <em>COCOON; Lecture Notes in
Computer Science (Oscar H. Ibarra and Louxin Zhang, editors),</em>
2387:391-400, 2002.</p>
</li>
<li>
<p>Oliver Eulenstein, Boris Mirkin, and Martin Vingron. Comparison of
annotating duplication, tree map­ping, and copying as methods to
compare gene trees with species trees. In <em>Mathematical Hierarchies
and Biology, DIMACS Series in Discrete Mathematics and Theoretical
Computer Science (Boris Mirkin, Fed R. McMorris, Fred S. Roberts,
and Andrey Rzhetsky, ed.)</em>, 37:71-94, American Mathematical Society,
Providence, RI, 1997.</p>
</li>
<li>
<p>Yan P. Yuan, Oliver Eulenstein, Martin Vingron, and Peer Bork.
Towards detection of sequence orthologs. In <em>German Conference on
Bioinformatics</em>, pages 25-30, 1997.</p>
</li>
<li>
<p>Oliver Eulenstein and Martin Vingron. On the Equivalence of Two Tree
Mapping Measures. In <em>Proceedings of the 20th Annual Conference of
the Classification and Knowledge Organization e.V. (R<strong>ü</strong>diger Klar
and Otto Opitz, ed.)</em>, University of Freiburg, Springer, Berlin,
March 6-8, 642-649:1996.</p>
</li>
<li>
<p>Oliver Eulenstein. Multi-viewing of an EDIF Netlist Schema.
Proceedings Fourth European EDIF Forum, Warrington, England, 11--12
Oct. 1990.</p>
</li>
</ol>
<p>[Book Chapters [5 chapters]]{.smallcaps}</p>
<ol>
<li>
<p>Oliver Eulenstein, S. Huzurbazar, and David Liberles. Reconciling
phylogenetic trees. In <em>Evolution after gene duplication (S.
Huzurbazar and D. Liberles, ed.),</em> Wiley, New York, pages
185-206, 2010.</p>
</li>
<li>
<p>Michael J. Sanderson, Cécile Ané, Oliver Eulenstein, David
Fernández-Baca, Kim Junhyong, Michelle M. McMahon, and Raul
Piaggio-Talice. Fragmentation of large data sets in phylogenetic
analyses. Chapter 7 (p. 199-216) in <em>Reconstructing Evolution
(Olivier Gascuel and Mike Steel, ed.),</em> Oxford University
Press, 2007.</p>
</li>
<li>
<p>Oliver Eulenstein. Consensus trees and supertrees. In <em>Handbook of
Computational Molecular Biology (Srinivas Aluru, ed.),</em>
Hall/CRC, 2005.</p>
</li>
<li>
<p>J. Gordon Burleigh, Oliver Eulenstein, David Fernández-Baca, and
Michael J. Sanderson. MRF super­trees. In <em>Phylogenetic supertrees:
combining information to reveal the tree of life (Olaf
Bininda-Emonds, ed.),</em> pages 65-86, Kluwer, 2004.</p>
</li>
<li>
<p>Raul Piaggio-Talice, J. Gordon Burleigh, and Oliver Eulenstein.
Quartet supertrees. In <em>Phylogenetic super­trees: combining
information to reveal the Tree of Life (Olaf Bininda-Emonds, ed.),</em>
pages 173-192, Kluwer, 2004.</p>
</li>
</ol>
